layer.go

```diff
@@ -196,6 +203,9 @@ func (l *Layer) Reader() (ReadAtCloser, error) {
 	if !l.init {
 		return nil, errors.New("claircore: unable to return Reader: uninitialized Layer")
 	}
+	if l.rd == nil {
```

Comment:
Yeah, was going to mention this, maybe it's time to start deprecating it (future).